### DISCUSSION TITLE
Yazı tura oyunu sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#311] [Soru Cevap]
Emrah ve Mahsun yazı tura oyunu oynarlar. Aynı parayı emrah 10 kez, mahsun ise 9 kez atacaktır.
Emrahın daha fazla tura atma olasılığı nedir?

**Comment by ElifNurSoyler** [comment]
```
e=10
m=9
eo=e*(1/2)
mo=m*(1/2)
print(eo-mo)
```

**Comment by yigidi** [comment]
10/2=5
9/2=4.5
5-4.5=0.5
0.5=%20

**Reply by sonsuzus** [reply]
Çözülmemiş sorulara yönlenin.
